workflows/prognostic_c48_run/runtime/names.py

```diff
@@ -20,3 +25,16 @@
     "dQu": EAST_WIND,
     "dQv": NORTH_WIND,
 }
+STATE_NAME_TO_TENDENCY = {value: key for key, value in TENDENCY_TO_STATE_NAME.items()}
+
+
+def is_state_update_variable(key, state: State):
+    if key in state.keys() and key not in TENDENCY_TO_STATE_NAME:
+        # the second check is to exclude derived variables such as dQu,v
+        return True
+    elif key == TOTAL_PRECIP_RATE:
```

Comment:
Does this need a special case here and one in the derived mapping alternate keys, or could it be done with only one of those?

Reply:
Just here, since this PR handles the accumulation from rate conversion independently of the derived mapping. i.e. the loop explicitly replaces this prediction with its equivalent accumulation.

Reply:
Oh, what I meant was that there are two special cases in this PR (the alternate keys in the derived mapper being the other), and whether we could have gotten away with just one.